rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
```

Comment:
What difference would it make? Will this execute a job against all repositories for every PR to one of those repositories right now? Or will it only execute against the repository a PR is requested against.